Polygon-Abstract/src/polygon/Triangle.java

```diff
@@ -0,0 +1,74 @@
+package polygon;
+
+import java.util.ArrayList;
+//import java.awt.Point;
+
+public class Triangle extends Polygon{
+
+	
+	public static final int NUMOFSIDES = 3;
+	
+	
+	
```

Comment:
I would like you to write a constructor `public Triangle()`. That was part of my design, but it is fine since you implemented the `Triangle(ArrayList<Point> coors)` perfectly.